Modules/AppointmentSlots/Resources/views/select_appointments/interview_schedule_error_message.blade.php

```diff
@@ -0,0 +1,19 @@
+@extends('appointmentslots::layouts.master')
+
+@section('content')
+
+<div>
+    <div style = "text-align: center;">
+        <svg style = "width:44; height:44; color:#d44950">{!! file_get_contents(public_path('icons/exclamation-circle.svg')) !!}</svg>
+    </div>
+    <h1 class="error-heading" style ="text-align: center;">Page Not found</h1><br>

```

Comment:
Avoid using `style` attribute in blade file
```suggestion
    <h1 class="error-heading text-center">Page Not found</h1><br>

```